build.gradle

```diff
@@ -98,19 +98,32 @@ javafx {
 
 def JLINK_DIR = "$buildDir/rcv"
 // Necessary to explicitly include due to https://github.com/BrightSpots/rcv/issues/625
-def MODULES_TO_ADD = "com.ctc.wstx"
+def MODULE_TO_ADD = "com.ctc.wstx"
```

Comment:
Nit: I lean towards keeping this as-is since it matches the option `add-modules`, and it also hints that we can add more if necessary.

Reply:
Agreed. For some reason, I thought the modules would be space-separated rather than comma-separated, so I thought only one module was allowed. I'll update the comment for future travelers.